source/intro.rst

```diff
@@ -65,8 +65,8 @@ Searching
 
 Since "go" is such a common English word, it is completely useless as a search
 term when googling.  Instead search for "golang".  It is standard practice in
-the community that all Go-related blog posts, Github repos, tweets, job
-postings, etc be tagged with "golang".
+the community that all Go-related blog posts, GitHub repos, tweets, job
+postings, etc., be tagged with "golang".
```

Comment:
Use of a period after "etc" is a stylistic matter of some contention.  In the English dialects I speak - Rustbelt American and Silicon Valley American - as well as in standard British usage, the period is considered superfluous.  

Please remove this change.